.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  release:
+    types: [prereleased]
+
+jobs:
+  create-ticket:
+    runs-on: ubuntu-latest
+    name: Create a CCB ticket in Jira
+    steps:
+      - name: Generate body
+        env:
+          # @see https://docs.github.com/en/rest/reference/releases#get-a-release
+          GITHUB_RELEASE_BODY: ${{ github.event.release.body }}
+          GITHUB_RELEASE_NAME: ${{ github.event.release.name }}
+          GITHUB_ACTIONS_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        run: export JIRA_DESCRIPTION="$(${GITHUB_WORKSPACE}/.github/workflows/create-ccb-ticket.sh)"
+        shell: bash
+      - name: Login to Jira
+        uses: acquia/gajira-login-2@server
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+      - name: Create CCB ticket
+        uses: acquia/gajira-create@server
+        with:
+          project: DX
+          issuetype: Release
+          summary: ${{ github.events.issue.title }}
+          description: ${{ env.JIRA_DESCRIPTION }}
+          # @todo Replace with CLI component ID.
+          fields: '{"components": [{"id": "21989"}]}'
```

Comment:
@danepowell Do you know the right ID for ACLI?

Reply:
I'm not Dane but I can help you with that. It's at `<JIRA_BASE_URL>/rest/api/latest/project/CLI`